### ISSUE TITLE
Unable to install this package

### ISSUE THREAD

**Issue by NxRx** [gbrlsnchs/jwt#48]
:~$ go version
go version go1.14.3 linux/amd64
:~$ go get -u github.com/gbrlsnchs/jwt/v3
package github.com/gbrlsnchs/jwt/v3: cannot find package "github.com/gbrlsnchs/jwt/v3" in any of:
        /usr/lib/newgo/go/src/github.com/gbrlsnchs/jwt/v3 (from $GOROOT)

-------
:~$ GO111MODULE=on
:~$ go get -u github.com/gbrlsnchs/jwt/v3
package github.com/gbrlsnchs/jwt/v3: cannot find package "github.com/gbrlsnchs/jwt/v3" in any of:
        /usr/lib/newgo/go/src/github.com/gbrlsnchs/jwt/v3 (from $GOROOT)

**Comment by joshartman** [comment]
Same issue.

**Comment by joshartman** [comment]
> Same issue.

Had to add the GO111MODULE setting, then it works.

**Comment by huddlestonka** [comment]
Same issue with go1.15.6 windows/amd64.

GO111MODULE=on is making no difference.

**Comment by gbrlsnchs** [comment]
I could not reproduce those errors, which seem to be much more user errors than lib errors.

**Comment by Mark90** [comment]
@joshartman  thanks, the GO111MODULE setting resolved it for me under Go 1.13.8  / Ubuntu 20.04

```
➜  repo git:(master)  ✗ go get -u github.com/gbrlsnchs/jwt/v3
package github.com/gbrlsnchs/jwt/v3: cannot find package "github.com/gbrlsnchs/jwt/v3" in any of:
        /usr/lib/go-1.13/src/github.com/gbrlsnchs/jwt/v3 (from $GOROOT)
        /home/mark/go/src/github.com/gbrlsnchs/jwt/v3 (from $GOPATH)
 ✗ go version
go version go1.13.8 linux/amd64
➜  repo git:(master) ✗ export GO111MODULE=on
➜  repo git:(master) ✗ go get -u github.com/gbrlsnchs/jwt/v3
go: finding github.com/gbrlsnchs/jwt v1.1.0
go: finding github.com/gbrlsnchs/jwt/v3 v3.0.0
go: downloading github.com/gbrlsnchs/jwt/v3 v3.0.0
go: downloading github.com/gbrlsnchs/jwt v1.1.0
go: extracting github.com/gbrlsnchs/jwt v1.1.0
go: extracting github.com/gbrlsnchs/jwt/v3 v3.0.0
go: downloading github.com/magefile/mage v1.9.0
go: downloading golang.org/x/crypto v0.0.0-20190927123631-a832865fa7ad
go: downloading golang.org/x/xerrors v0.0.0-20191204190536-9bdfabe68543
go: extracting golang.org/x/xerrors v0.0.0-20191204190536-9bdfabe68543
go: extracting golang.org/x/crypto v0.0.0-20190927123631-a832865fa7ad
go: extracting github.com/magefile/mage v1.9.0
go: finding github.com/magefile/mage v1.11.0
go: finding golang.org/x/xerrors latest
go: finding golang.org/x/crypto latest
go: downloading github.com/magefile/mage v1.11.0
go: downloading golang.org/x/crypto v0.0.0-20210616213533-5ff15b29337e
go: extracting golang.org/x/crypto v0.0.0-20210616213533-5ff15b29337e
go: extracting github.com/magefile/mage v1.11.0
➜  repo git:(master) ✗
```
